MapboxDirections/MBVoiceInstruction.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+@objc(MBVoiceInstruction)
+public class VoiceInstruction: NSObject, NSSecureCoding {
+    
+    public let distanceAlongGeometry: CLLocationDirection
```

Comment:
This library has generally avoided the term “geometry” as a countable noun. Anyhow, I think it would be less ambiguous to call it `distanceAlongStep`, since routes also have geometry data attached to them.